### DISCUSSION TITLE
make Z Seam more visible in Gcode View

### DISCUSSION THREAD

**Discussion by nickeeey** [supermerill/SuperSlicer#2364] [Community contribution]
Hi, i switch between different Z Seam Settings and i loved it from cura to highlight the Z Seam. Can you also implement it?

**Comment by supermerill** [comment]
implemented (by prusa) in the 2.4 beta

**Comment by lorroi** [comment]
Where can I find this? I am using 2.4, but I can't seem to find the proper menu :)

**Reply by supermerill** [reply]
![image](https://user-images.githubusercontent.com/6536403/156260051-e3f31b5d-ace4-4c5b-ab91-ae58c6b22879.png)
